aws/data_source_aws_instance.go

```diff
@@ -208,6 +208,11 @@ func dataSourceAwsInstance() *schema.Resource {
 							Computed: true,
 						},
 
+						"throughput": {
```

Comment:
Documentation missing in `website/docs/d/instance.html.markdown` -- will followup post-merge. 👍